rerun_py/rerun_sdk/rerun/log/__init__.py

```diff
@@ -3,29 +3,6 @@
 import numpy as np
 import numpy.typing as npt
 
-from rerun import bindings
-
-__all__ = [
```

Comment:
we can bring that one back, but it'd be nice if it listed stuff that actually exists...